woocommerce/wc-admin
# Fix/7174 dismiss button alignment

Fixes #7174

Add global (under `body.woocommerce-page` anyway) style rule to make buttons take the full width of ellipsis menu popovers and center their labels.

### Screenshots

Before:

![Screen Shot 2022-01-13 at 13 45 56](https://user-images.githubusercontent.com/890809/149398764-b5fe9680-007e-4c79-a66e-b57220d815f9.png)

After:

![Screen Shot 2022-01-19 at 10 48 06](https://user-images.githubusercontent.com/890809/150176262-3f8de65b-2220-41ed-af31-1f2b15f8807b.png)

![Screen Shot 2022-01-19 at 10 48 23](https://user-images.githubusercontent.com/890809/150176293-5eef7762-50ac-4458-888b-4ffffee638d8.png)

### Detailed test instructions:

-   Complete OBW and select Google Listings and Ads in marketing step.
-   Navigate to WooCommerce Home.
-   Click the vertical dot menu in the "Set Up Google Listings & Ads" and check that the Dismiss button is centered in the menu container.
